### PR TITLE
Use TabLayout and ViewPager for about fragments; closes #85

### LINKED ISSUE #85: About: Make tabs swipeable
Currently tabs in the about-section are only clickable. One should be able to switch the tab by swiping left/right.

### PR DESCRIPTION
**Pull request by fxedel**
With this views, the TabLayout (which holds the three tab buttons) can easily be connected to the ViewPager (which holds the three tab fragments). That allows you to swipe the tabs.

### CODE REVIEW

**Comment by stefan-niedermann** [comment]
Thanks for your contribution :) I'll review this in the next view days, please be patient. Looks good :+1:

File: app/src/main/java/it/niedermann/owncloud/notes/android/activity/AboutActivity.java [hunk 1]
```diff
@@ -1,30 +1,78 @@
...
+        public TabsPagerAdapter(FragmentManager fragmentManager) {
+            super(fragmentManager);
+        }
+
+        @Override
+        public int getCount() {
+            return PAGE_COUNT;
+        }
+
+        // return the right fragment for the given position
```
Comment (stefan-niedermann) [inline app/src/main/java/it/niedermann/owncloud/notes/android/activity/AboutActivity.java]: Use JavaDoc syntax

File: app/src/main/java/it/niedermann/owncloud/notes/android/activity/AboutActivity.java [hunk 2]
```diff
@@ -1,30 +1,78 @@
...
+
+                case 2:
+                    return new AboutFragmentLicenseTab();
+
+                default:
+                    return null;
+            }
+        }
+
+        // generate title based on given position
```
Comment (stefan-niedermann) [inline app/src/main/java/it/niedermann/owncloud/notes/android/activity/AboutActivity.java]: Use JavaDoc syntax

**Comment by stefan-niedermann** [comment]
Fixed with https://github.com/stefan-niedermann/OwnCloud-Notes/commit/4baf533bc1ad61a15d018dc980e1178b40584460, thanks!

**Comment by stefan-niedermann** [comment]
@fxedel feel free to add yourself to the "developer"-section in the about-page and set up a PR. (think about every language file)

**Comment by fxedel** [comment]
Great, thanks! Have a look at #110 please :)

**Pull request closed without merging**